### PR TITLE
BHBC-2251: Dataset details update

### PR DESCRIPTION
**Pull request by AlfredRosenthal**
# Overview

## Links to Jira tickets

[BHBC-2251](https://quartech.atlassian.net/browse/BHBC-2251?atlOrigin=eyJpIjoiYTM5ZTE1ZTg3NmU4NGY3ODhhODBkYWMwMzQ0YTU3Y2UiLCJwIjoiaiJ9)

## Description of relevant changes

- Modified dataset page for Survey/ Project specific fields
- Updated CSS to be more inline with tickets UX pin
- Details section only

## PR Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### Code

- [ ] New files/classes/functions have appropriately descriptive names and comment blocks to describe their use/behaviour
- [ ] I have avoided duplicating code when possible, moving re-usable pieces into functions
- [ ] I have avoided hard-coding values where possible and moved any re-usable constants to a constants file
- [ ] My code is as flat as possible (avoids deeply nested if/else blocks, promise chains, etc)
- [ ] My code changes account for null/undefined values and handle errors appropriately
- [ ] My code uses types/interfaces to help describe values/parameters/etc, help ensure type safety, and improve readability

### Style

- [ ] My code follows the established style conventions
- [ ] My code uses native material-ui components/icons/conventions when possible

### Documentation

- [ ] I have commented my code sufficiently, such that an unfamiliar developer could understand my code
- [ ] I have added/updated README's and related documentation, as needed

### Tests

- [ ] I have added/updated unit tests for any code I've added/updated
- [ ] I have added/updated the Postman requests/tests to account for any API endpoints I've added/updated

### Linting/Formatting

- [ ] I have run the linter and fixed any issues, as needed  
       _See the `lint` commands in package.json_
- [ ] I have run the formatter and fixed any issues, as needed  
       _See the `format` commands in package.json_

### SonarCloud

- [ ] I have addressed all SonarCloud Bugs, Vulnerabilities, Security Hotspots, and Code Smells

### CODE REVIEW

File: app/src/hooks/templates/SIMS-handlebar-template.tsx
Comment (curtisupshall) [inline app/src/hooks/templates/SIMS-handlebar-template.tsx:1]: - Should we move the `templates` folder from `src/hooks/templates` to just `src/templates`?
 - This isn't a React file, so it should just have a `.ts` extension

File: app/src/hooks/api/useDatasetApi.ts [hunk 1]
```diff
@@ -55,11 +56,23 @@ const useDatasetApi = (axios: AxiosInstance) => {
...
   };
 
+  /**
+   * Fetch the signed handlebar template for a given dataset ID.
+   *
+   * @param {string} datasetId
+   * @return {*}  {Promise<string>}
+   */
+  const getHandlebarsTemplate = async (datasetId: string): Promise<string> => {
+    // const { data } = await axios.get<string>(`api/dwc/submission/${datasetId}/template`);
```
Comment (curtisupshall) [inline app/src/hooks/api/useDatasetApi.ts]: Nit: Remove commented code

File: app/src/utils/handlebarsUtils.ts [hunk 2]
```diff
@@ -34,6 +34,15 @@ export const useHandlebars = () => {
     });
   };
 
+  const capitalizeFirst = () => {
+    Handlebars.registerHelper('capFirst', (text) => {
+      if (typeof text === 'string') {
+        text = text.toLowerCase();
+        text = text.charAt(0).toUpperCase() + text.slice(1);
```
Comment (curtisupshall) [inline app/src/utils/handlebarsUtils.ts]: Nit: template strings behave more reliably than string append:
```
if (typeof text === 'string') {
  return `${text.charAt(0).toUpperCase()}${text.slice(1).toLowerCase()}`;
}

return text;
```

**Review by curtisupshall — COMMENTED** [review]
Tested and works as expected! Just had a couple comments on the code

File: app/src/utils/handlebarsUtils.ts (hunk 2 above)
Comment (AlfredRosenthal) [reply app/src/utils/handlebarsUtils.ts]: good call, updated accordingly

File: app/src/hooks/api/useDatasetApi.ts (hunk 1 above)
Comment (AlfredRosenthal) [reply app/src/hooks/api/useDatasetApi.ts]: done

**Review by curtisupshall — DISMISSED** [review]
Code looks good!

**Comment by curtisupshall** [comment]
> Code looks good!

Tested again locally, works as expected.

**Review by NickPhura — DISMISSED** [review]

**Review by anissa-agahchen — APPROVED** [review]

**Review by NickPhura — APPROVED** [review]

**Pull request merged**